src/Compilers/CSharp/Portable/Symbols/TupleFieldSymbol.cs

```diff
@@ -58,17 +58,17 @@ internal static FieldSymbol GetUnderlyingField(int containingTupleArity, NamedTy
                 containingLinkSize = tupleRemainder;
             }
 
+            NamedTypeSymbol linkType = TupleTypeSymbol.GetNestedTupleType(containingUnderlyingType, fieldChainLength - 1);
+
             // PROTOTYPE(tuples) Constructing a TupleTypeSymbol with TupleFieldSymbols should not produce diagnostics for missing well-known members and type members.
             //                      The error handling should be shifted to lowering. This will help when we implement metadata loading.
             WellKnownMember wellKnownTypeMember = TupleTypeSymbol.GetTupleTypeMember(containingLinkSize, fieldRemainder);
-            var linkField = (FieldSymbol)Binder.GetWellKnownTypeMember(binder.Compilation, wellKnownTypeMember, diagnostics, syntax: syntax);
+            var linkField = (FieldSymbol)Binder.GetWellKnownTypeMember(binder.Compilation, wellKnownTypeMember, diagnostics, type: linkType.OriginalDefinition, syntax: syntax);
```

Comment:
As it stands right now, the change doesn't get us where we want to be. We should be able to create of a tuple type symbol during metadata import, this should happen without presence of compilation and should not produce any errors. The same should be true when we are creating tuple type symbol in source. The only place to report errors is during lowering and we can do this there very easy without re-purposing any existing helpers in Binder, which makes it very easy for people to make a usage mistake. Let's discuss this in more details tomorrow morning.